src/OrchardCore/OrchardCore.Workflows.Abstractions/Models/WorkflowType.cs

```diff
@@ -1,11 +1,12 @@
 using System.Collections.Generic;
+using OrchardCore.Entities;
 
 namespace OrchardCore.Workflows.Models
 {
     /// <summary>
     /// Represents a workflow type.
     /// </summary>
-    public class WorkflowType
+    public class WorkflowType : Entity
```

Comment:
Maybe `IEntity` would have been sufficient

Reply:
@jtkech `IEntity` would have been sufficient but then you would have to implement it. `Entity` provides a default implementation to `IEntity`

Reply:
Okay then